tests/integration/ha/test_backups.py

```diff
@@ -240,7 +240,7 @@ async def test_create_backup_and_restore(
             leader_id,
             unit_ip=unit_ip,
         )
-    ) > 0
+    ) != ""
```

Comment:
I would change this to the following, so that it throws an exception if it's not a date format or if the date is wrong:
```
from datetime import datetime

date_before_backup = datetime.now()

backup_id = await create_backup(ops_test, leader_id, unit_ip=unit_ip)
assert datetime.strptime(backup_id, OPENSEARCH_BACKUP_ID_FORMAT) > date_before_backup
```